libs/sr/page-team/src/BuildsDisplay.tsx

```
@@ -0,0 +1,195 @@
+function BuildBase({
+  name,
+  description,
+  buildGrid,
+  active,
+  onClick,
+}: {
+  name: string
+  description?: string
+  buildGrid: ReactNode
+  active?: boolean
+  onClick?: () => void | true
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)